plugins/CoreHome/Columns/IdSite.php

```diff
@@ -51,4 +57,14 @@ public function onAnyGoalConversion(Request $request, Visitor $visitor, $action)
     {
         return $request->getIdSite();
     }
-}
\ No newline at end of file
+
+    public function formatValue($value, $idSite, Formatter $formatter)
+    {
+        try {
+            $siteName = Site::getNameFor($value);
+            return $siteName . ' (ID ' . $value . ')';
+        } catch (\Exception $ex) {
+            return parent::formatValue($value, $idSite, $formatter);
```

Comment:
When a site was deleted and name cannot be resolved, does it make sense to format like "Site ID: XYZ"?

Reply:
Sure, can do this.